site/app/libraries/database/DatabaseQueriesPostgresql.php

```diff
@@ -814,37 +835,34 @@ public function updateActiveVersion($g_id, $user_id, $team_id, $version) {
         }  
     }
 
+    public function insertGradeableData(Gradeable $gradeable) {
+        $params = array($gradeable->getId(), $gradeable->getUser()->getId(),
+                        $gradeable->getOverallComment(), $gradeable->getStatus(), 0,
+                        $gradeable->getActiveVersion());
+        $this->database->query("INSERT INTO 
+gradeable_data (g_id, gd_user_id, gd_overall_comment, gd_status, gd_late_days_used, gd_active_version)
+VALUES (?, ?, ?, ?, ?, ?)", $params);
+        return $this->database->getLastInsertId("gradeable_data_gd_id_seq");
+    }
+
     public function updateGradeableData(Gradeable $gradeable) {
```

Comment:
Discussed this today, I'm leaving it in as it'll eventually be used (when we are submitting electronic gradeable rubrics where the gradeable_data table will actually warrant a change and this function can be properly tested), just not at this exact moment.